libs/design-system/icon-button/src/lib/icon-button-styles/icon-button-styles.component.scss

```diff
@@ -1,16 +1,31 @@
-button[mat-icon-button] {
+button[mat-icon-button],
+a[mat-icon-button] {
   --mat-icon-button-hover-state-layer-opacity: 0.08;
   --mat-icon-button-pressed-state-layer-opacity: 0.16;
-  --mat-icon-button-state-layer-color: var(--sys-secondary);
   --mat-icon-button-focus-state-layer-opacity: 0;
+  --mdc-icon-button-icon-color: var(--sys-on-tertiary-fixed);
 
   &:focus-visible {
-    outline: 2px solid var(--sys-tertiary);
+    --mdc-icon-button-icon-color: var(--sys-secondary);
+    border: 2px solid var(--sys-tertiary);
```

Comment:
Please change back to using outline. Using border causes a layout shift making the icon move slightly